exercises/sieve/package.yaml

```diff
@@ -0,0 +1,20 @@
+name: sieve
+
+dependencies:
+  - base
+
+library:
+  exposed-modules: Sieve
+  source-dirs: src
+  dependencies:
+    - vector
```

Comment:
This leads me to see an interesting point! (This may apply to exercises other than sieve as well)

This `vector` is here because the example solution uses it, right? It is not necessarily the case that every submitted solution will use it. 

Are we OK with leaving them in here?

Then again, the alternative is to have some custom thing that splices in the example solution's dependencies when we go to Travis, which is not necessarily something we want to tackle.

So our answer might be "We're OK with it", as long as students don't think their solution must use `vector`.

My personal answer is probably to keep it as is for now, see whether any problems come up